app/src/editor/canvas/utils/isEditableElement.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * True if element is a form control or editable element.
+ */
+
+export default function isEditableElement(element) {
```

Comment:
I guess this is more generic, works for anything. I didn't prematurely promote it to `$shared` but it could easily go there as it has nothing really to do with canvases or even anything streamr specific. I've used similar functionality a number of times before in other apps to prevent keyboard handlers misfiring. Easy to forget events bubble and accidentally handle a key event that was intended for an input or contenteditable.